src/routes/landing/containers/LandingContainer.js

```diff
@@ -48,28 +47,23 @@ const mapStateToProps = (state: StateType): StatePropsType => {
 
 const mapDispatchToProps = (dispatch: Dispatch<StoreActionType>, ownProps: OwnPropsType): DispatchPropsType => {
   return {
-    fetchCities: () => dispatch({
-      type: 'FETCH_CITIES'
-    }),
     navigateToDashboard: (cityCode: string, language: string) => {
       const path = `/${cityCode}/${language}`
       const key: string = generateKey()
 
-      const action: NavigationReplaceAction = StackActions.replace({
+      const action = NavigationActions.navigate({
```

Comment:
Revert this change; We need to set newKey here and reset the stack:

We need react-navigation to give this route the very same key that we generate above. (Otherwise there's some strange default key, which messes things up)

Reply:
Very strange, but okay :D Thanks :)